help_to_heat/frontdoor/views.py

```diff
@@ -149,26 +149,38 @@ def change_page_view(request, session_id, page_name):
     return page_map[page_name](request, session_id, page_name, is_change_page=True)
 
 
-def get_prev_next_page_name(page_name):
-    if page_name in schemas.page_prev_next_map:
-        prev_page_name = schemas.page_prev_next_map[page_name]["prev"]
-        next_page_name = schemas.page_prev_next_map[page_name]["next"]
+def get_prev_next_page_name(page_name, session_id=None):
```

Comment:
Main changes are here; it switches between two flows (?)

Reply:
I think what I wrote here is.. hacky 🤦

Reply:
I think it's not ideal, but I suspect it will be replaced when we implement PC-498. Do you agree?

Reply:
I hope so! I hope it's not too hacky to make it impossible to implement PC-498 (?) 😅